src/lando/utils/management/commands/create_environment_repos.py

```diff
@@ -252,6 +252,15 @@
             "milestone_tracking_flag_template": "cf_status_firefox_esr{milestone}",
             "commit_flags": [DONTBUILD],
         },
+        {
```

Comment:
We define these repos as dicts, then when using them we do `Repo.objects.create(**definition)`. Perhaps it would make more sense to define these as `Repo` and then create them with `repo.save()` instead.

Reply:
On production this is basically a single use script. On other environments, it will be reused so it is worth improving it some more if we feel the need. I think this is pretty good as is for the time being, but we can consider these improvements for the future.